Dockerfile

```diff
@@ -10,9 +10,9 @@ RUN apk add --no-cache $BUILD_PACKAGES $DEV_PACKAGES $RUBY_PACKAGES
 
 # install rubygem
 COPY Gemfile Gemfile.lock ./
-COPY vendor/cache ./vendor/cache
-COPY node_modules ./node_modules
-COPY public/assets ./public/assets
+# COPY vendor/cache ./vendor/cache
+# COPY node_modules ./node_modules
+# COPY public/assets ./public/assets
```

Comment:
I think you forgot to uncomment those lines :)

Also, `COPY vendor/cache` is enough, the two other lines are useless